composer.json

```diff
@@ -17,11 +17,17 @@
         "dhii/php-cs-fixer-config": "dev-php-5.3",
         "phpunit/phpunit": "^4.8",
         "ptrofimov/xpmock": "^1.1",
-        "codeclimate/php-test-reporter": "<=0.3.2"
+        "codeclimate/php-test-reporter": "<=0.3.2",
+        "dhii/validation-interface": "^0.1"
```

Comment:
Because you're consuming its interfaces in your code. If developer dependencies are not installed, the references to the interfaces won't be resolved.